lumicks/pylake/detail/widefield.py

```diff
@@ -193,6 +193,11 @@ class ImageDescription:
     def __init__(self, tiff_file, align_requested):
         first_page = tiff_file.pages[0]
         self.is_rgb = first_page.tags["SamplesPerPixel"].value == 3
+        self._cmap = (
+            {"red": 0, "green": 1, "blue": 2, "rgb": slice(None), None: slice(None)}
+            if self.is_rgb
+            else {None: slice(None)}
```

Comment:
Can't this just be an empty dict?

Or even just `None`? (Considering you have the if-statement down there anyway and alignment matrices only exist in multi-color images?).

Reply:
agreed. and having the axes in the correct order, the reasons for making a more complicated handling basically went away, so we're back to the original code for this